.drone.star

```diff
@@ -126,6 +126,80 @@ def testing(ctx):
           },
         },
       },
+      {
+        'name': 'ldap',
```

Comment:
Running this as a step doesn't make sense.

Reply:
I've opened an issue to fix it https://github.com/owncloud/ocis-reva/issues/70